tests/test_meshtastic_utils_async.py

```
@@ -0,0 +1,621 @@
+            _fire_and_forget(_noop())
+
+    def test_fire_and_forget_ignores_cancelled_error(self):
+        """Ensure fire-and-forget ignores CancelledError in callbacks."""
+        from mmrelay.meshtastic_utils import _fire_and_forget
+
+        async def _noop():
+            return None
+
+        fake_task = self._ExceptionTask(raise_exc=asyncio.CancelledError())
+
+        def _submit(coro, loop=None) -> Any:
```

Ruff (0.15.12)

[warning] Unused function argument: `loop`

(ARG001)